ecology.yaml

```diff
@@ -895,6 +895,10 @@ tools:
     owner: ecology
     tool_panel_section_label: 'Get Data'
 
+  - name: aquainfra
```

Comment:
Oo, was that really the tool ID? This is way to generic :(

Reply:
Yeah, I also thought that this morning, but I thought it's too late to change that...

Reply:
@bgruening  Shall I just update the tool with a more specific id?

Reply:
until now its not installled please change that ...